components/PageNavigation.vue

```diff
@@ -26,12 +26,35 @@
         v-if="isAuthenticated"
         class="nav-item"
       >
-        <b-link
-          :to="localePath({ name: 'account' })"
+        <b-dropdown
+          right
+          no-caret
+          variant="white"
           class="nav-link"
+          :class="isAccountPage && 'exact-active-link'"
         >
-          <span>{{ $t('account.linkAccount') }}</span>
-        </b-link>
+          <template
+            slot="button-content"
+          >
+            <span class="label">{{ $t('account.linkAccount') }}</span>
+          </template>
+          <b-dropdown-item
+            :to="localePath({ name: 'account' })"
+          >
+            <span class="label">{{ $t('account.profile') }}</span>
+          </b-dropdown-item>
+          <b-dropdown-item
+            :href="accountSettingsPage"
+          >
+            <span class="label">{{ $t('account.settings') }}</span>
+          </b-dropdown-item>
+          <b-dropdown-divider />
+          <b-dropdown-item
+            :to="localePath({ name: 'account-logout' })"
```

Comment:
I have just realised that the account logout URL should not be localised. Could you make that change with this patch:
```diff
diff --git a/components/PageNavigation.vue b/components/PageNavigation.vue
index 544c6e12..93dfa9cb 100644
--- a/components/PageNavigation.vue
+++ b/components/PageNavigation.vue
@@ -50,7 +50,7 @@
           </b-dropdown-item>
           <b-dropdown-divider />
           <b-dropdown-item
-            :to="localePath({ name: 'account-logout' })"
+            :to="{ name: 'account-logout' }"
           >
             <span class="label">{{ $t('account.linkLogout') }}</span>
           </b-dropdown-item>
diff --git a/middleware/l10n.js b/middleware/l10n.js
index 58c12363..7d0f19af 100644
--- a/middleware/l10n.js
+++ b/middleware/l10n.js
@@ -18,7 +18,11 @@ function appSupportsLocale(locale) {
 
 export default ({ app, route, redirect, req }) => {
   // Exit early if this is an auth callback
-  if (app.$auth && (route.path === app.$auth.options.redirect.callback || route.path === app.$auth.options.redirect.login)) return;
+  if (app.$auth && [
+    app.$auth.options.redirect.callback,
+    app.$auth.options.redirect.login,
+    app.$auth.options.redirect.logout
+  ].includes(route.path)) return;
 
   // Is there a locale in the URL path already?
   const routePathLocaleMatch = route.path.match(/^\/([a-z]{2})(\/.*)?$/);
diff --git a/nuxt.config.js b/nuxt.config.js
index faa82fe0..9f7345cb 100644
--- a/nuxt.config.js
+++ b/nuxt.config.js
@@ -144,11 +144,12 @@ const config = {
         silentFallbackWarn: true,
         dateTimeFormats: i18nDateTime
       },
-      // Disable redirects to account callback & login pages
+      // Disable redirects to account pages
       parsePages: false,
       pages: {
         'account/callback': false,
-        'account/login': false
+        'account/login': false,
+        'account/logout': false
       },
       // Enable browser language detection to automatically redirect user
       // to their preferred language as they visit your app for the first time
@@ -232,7 +233,7 @@ if (Number(process.env['ENABLE_XX_USER_AUTH'])) {
     // 'home' option for redirection after login
     redirect: {
       login: '/account/login',
-      logout: '/',
+      logout: '/account/logout',
       callback: '/account/callback',
       home: '/account'
     },
```